.gitignore

```diff
@@ -39,3 +39,7 @@ node_modules
 website/resources
 website/public
 website/.hugo_build.lock
+
+# Files built for SELinux troubleshooting
+policy.33
+file_contexts
```

Comment:
how do they end up in working directory?

Reply:
I build policy locally for `audit2allow` purposes. They aren't created by build system normally, but I use those for development.

Reply:
let's put those under `hack` then if we ever want to debug

Reply:
we generate & commit those files